src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -91,7 +91,8 @@ const SampleChatScreen = () => {
       setLoading(true);
       await moveToSignedChannel({
         oldChannelId: selectedChannel?.id,
-        targetUserId: memberChat.user_id
+        targetUserId: memberChat.user_id,
+        source: 'userId'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `source` property is hardcoded as `'userId'`. This might limit the flexibility of your function. If the source can be different in other scenarios, consider passing it as a parameter to the function instead of hardcoding it.
```diff
-        source: 'userId'
+        source: someSourceVariable
```
If `'userId'` is the only possible value for `source`, then this change is fine.


<!-- This is an auto-generated comment by OSS CodeRabbit -->